index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Project 2: The News Site</title>
+  <link rel="stylesheet" href="./style.css" />
+</head>
+
+<body>
+  <!-- header  -->
+  <header>
+    <img class="logo" src="logo.png" alt="logo">
+    <h1 class="heading">World Times</h1>
+    <a class="first-a" href="#">LATEST NEWS</a>
+    <a class="first-a" href="#">SUBSCRIBE</a>
+    <a class="second-a" href="#">ACCOUNT</a>
+    <a class="second-a" href="#">🔍 SEARCH</a>
+  </header>
+  <p class="date">Monday, January 20, 2020</p>
+
+  <!-- The big news  -->
+  <section class="big-news">
+    <div>
+      <img src="Brexit.jpg" alt="Placeholder big">
+    </div>
+    <div>
+      <h2 class="heading">Brexit day: what happens after January 31?</h2>
+      <p class="text">Brexit day is fast approaching, with the UK on track to officially leave the European Union (EU)
+        in just a few
+        weeks time. But how did we get to this stage and what is in the Brexit deal?
+        Here is everything you need to know about the UK's departure, including what the deal means for UK citizens and
+        the changes to travel, investments and savings we can expect to see post-Brexit. </p>
+    </div>
+  </section>
+
+  <!-- opther-news  -->
+  <section class="other-news">
+    <a href="#">
+      <img src="trump.jpg" alt="Placeholder-small">
```

Comment:
It's better to keep the `alt` empty than placeholder text :)
```suggestion
      <img src="trump.jpg" alt="">
```